### ISSUE TITLE
[Feature] note-taking w/ local LLM @ ollama

### ISSUE THREAD

**Issue by dvorka** [dvorka/mindforger#1539]
**Enhancement / feature description:**

MindForger with local LLM support.

**Comment by dvorka** [comment]
![llama-chat-1st](https://github.com/dvorka/mindforger/assets/2090143/107d3ad0-acbb-4e22-b1ae-0db86fedee91)

**Comment by zeigerpuppy** [comment]
A question regarding status of this feature.

I see there is the option to set `* Wingman LLM provider:` in the `~/.mindforger.md` file.

Is there some more info of what needs to be configured there?
I have a local `ollama` installation already running

**Comment by dvorka** [comment]
@zeigerpuppy I apologize, but I didn't release the `ollama` integration yet - it's fully functional in the development branch, but I didn't finish configuration (dialog) yet. I'm using the feature myself and it works nicely - in other words I'm really close, I just need to make the setup "user friendly". I was pulled to my other project some time ago, but I will try to give finishing `ollama` priority. Thank you for asking about the feature!

**Comment by balcala** [comment]
Any update on this?

As a former Linux sys admin "user friendly" is not necessary, editing the ~/mindforger.md would work for me

I also have a degree in Cyber Security and I don't really want to send personal data to Open AI if I can avoid it

**Comment by johnblommers** [comment]
> editing the ~/mindforger.md would work for me

Agreed. Some documentation on manually configuring MindForger to use Ollama would go a long way to satisfying my need for data privacy.

**Comment by fredlim** [comment]
How is the update? I don't mind the manually configuration of ollama with a text editor without the GUI dialog.

**Comment by dvorka** [comment]
Implemented by #1558
